spec/services/benefit_check_service_spec.rb

```diff
@@ -57,6 +57,24 @@
           expect(check.benefits_valid).to be false
         end
       end
+
+      context 'simulating a 400 error' do
+        let(:message) { { 'error': "LSCBC210: Error in request parameter 'Surname'" }.to_json }
+
+        before do
+          stub_request(:post, "#{ENV['DWP_API_PROXY']}/api/benefit_checks").
+            to_return(status: 400, body: message, headers: {})
+          described_class.new(check)
+        end
+
+        it 'returns the error in message' do
```

Comment:
Maybe even worth simulating a 500 response in here with content you dont care about and probably even in HTML instead of JSON ?  This will prove that a 500 response (applies to others as well) will not break your code and will give you the error you want within your app ?

Reply:
There is a test covering the 500 already :)